src/components/CodeView/index.tsx

```diff
@@ -21,13 +21,16 @@ import {
   codeShouldBeTrimmed,
   gettext,
   getLanguageFromMimeType,
+  sendPerfTiming,
   shouldAllowSlowPages,
 } from '../../utils';
 import { Version } from '../../reducers/versions';
 import LinterProvider, { LinterProviderInfo } from '../LinterProvider';
 import GlobalLinterMessages from '../GlobalLinterMessages';
 import SlowPageAlert from '../SlowPageAlert';
 
+const { Profiler } = React;
```

Comment:
That's weird but why not. Can't you import it as usual? (or use `React.Profiler`)

Reply:
Sure, I can try the former and if that doesn't work use the latter. I cannot remember exactly why I needed to do this.